src/main/java/org/spongepowered/api/event/entity/living/humanoid/ChangeLevelEvent.java

```diff
@@ -25,13 +25,17 @@
 package org.spongepowered.api.event.entity.living.humanoid;
 
 import org.spongepowered.api.event.Cancellable;
+import org.spongepowered.api.event.entity.ChangeEntityExperienceEvent;
 import org.spongepowered.api.event.entity.living.humanoid.player.TargetPlayerEvent;
 import org.spongepowered.api.util.annotation.eventgen.GenerateFactoryMethod;
 
 /**
  * Called when a human's level is changed.
+ *
+ * @deprecated Use {@link ChangeEntityExperienceEvent} instead
  */
 @GenerateFactoryMethod
+@Deprecated
```

Comment:
Villagers can implement this as well tbh, since they do "level up".

Reply:
Unfortunately making this change would require changing the event and break it due to its support for Humanoids which Villagers are not. I think a better choice would be to introduce a new event to handle the villager's leveling up.